hazelcast/src/main/java/com/hazelcast/nio/ascii/TextReadHandler.java

```diff
@@ -210,6 +210,9 @@ private boolean isCommandTypeEnabled(TextCommand command) {
                 boolean isMancenterRequest = uri.startsWith(HttpCommandProcessor.URI_MANCENTER_CHANGE_URL);
                 boolean isClusterManagementRequest = uri.startsWith(HttpCommandProcessor.URI_CLUSTER_MANAGEMENT_BASE_URL);
                 boolean isHealthCheck = healthcheckEnabled && uri.startsWith(HttpCommandProcessor.URI_HEALTH_URL);
```

Comment:
>  I know we already have some form of validation here

Yes that is what led me to also put my check here. Your suggestion makes it a much cleaner change though! 

Thanks!